docs/ddl-of-carbondata.md

```diff
@@ -173,14 +174,24 @@ CarbonData DDL statements are documented here,which includes:
 
    - ##### Table Block Size Configuration
 
-     This command is for setting block size of this table, the default value is 1024 MB and supports a range of 1 MB to 2048 MB.
+     This property is for setting block size of this table, the default value is 1024 MB and supports a range of 1 MB to 2048 MB.
 
      ```
      TBLPROPERTIES ('TABLE_BLOCKSIZE'='512')
      ```
- 
+
      **NOTE:** 512 or 512M both are accepted.
 
+   - ##### Table Blocklet Size Configuration
+
+     This property is for setting blocklet size in the carbondata file, the default value is 64 MB.
+     Blocklet is the minimum IO read unit, so the smaller the value, the better it is for filtering queries.
```

Comment:
I feel this statement may mislead users because if we reduce IO unit size, number of IO operations become more and it hits the performance.  It would be better in case of point queries not all filter queries, so better mention point queries not all filter queries

Reply:
ok, fixed